commitlint.config.js

```diff
@@ -0,0 +1,14 @@
+module.exports = {
+  extends: ['@commitlint/config-angular'],
+  rules: {
+    'subject-max-length': [1, 'always', 72],
+    'scope-empty': [1, 'never'],
+    'scope-enum': [2, 'always', ['app', 'lib']],
+    'scope-case': [2, 'always', 'lowerCase'],
+    'type-enum': [
+      2,
+      'always',
+      ['build', 'ci', 'docs', 'feat', 'fix', 'perf', 'refactor', 'release', 'revert', 'style', 'test']
```

Comment:
This is missing `chore`. I am using this type for things like updating the README. Or would you use `docs` for that?

Reply:
I've also always used `chore` for these kind of commits. But apparently it's no more used: https://github.com/marionebl/commitlint/issues/146. Also the official Angular contribution guideline don't use it any more: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#type